lib/modules/manager/flux/readme.md

```diff
@@ -26,6 +27,12 @@ Renovate can update `git` references from `GitRepository` resources.
 
 The `flux` manager only updates `GitRepository` fields that have a `tag` or `commit` key.
 
+### OCIRepository support
+
+Renovate can update `oci` references from `OCIRepository` resources.
+
+The `flux` manager only updates `OCIRepository` fields that have a `digest` or `tag` key. If both exist `digest` takes precedence.
```

Comment:
```suggestion
The `flux` manager only updates `OCIRepository` fields that have a `digest` or `tag` key.
If `tag` _and_ `digest` exist then `digest` takes precedence.
```

We follow the "one sentence per line pattern" in our docs. Plus we should be explicit about the fields, so people don't have to guess what "both exist" means. 😉